2019/September/community_news.md

```diff
@@ -0,0 +1,47 @@
+# PythonPune Community Newsletter [September 2019 Edition]
+
+## News from Python Community
+
+* Python: Goodbye Print Statements, Hello Debugger! by Nina Zakharenko 
+  https://blog.adafruit.com/2019/08/08/python-goodbye-print-statements-hello-debugger-by-nnja-python-pythonhardware-debugging/
+
+* Sunsetting Python 2 
+  https://www.python.org/doc/sunset-python-2/
+
+* Vote to promote Joannah Nanjekye as a core dev 
+  https://discuss.python.org/t/vote-to-promote-joannah-nanjekye-as-a-core-dev/2347
+
+* Coverage.py 5.0a7, and the future of pytest-cov 
+  https://nedbatchelder.com/blog/201909/coveragepy_50a7_and_the_future_of_pytestcov.html
+
+* Introduction to Python asyncio for Hackers 
+  https://docs.google.com/presentation/d/1T1OhwqqkxfjhH8zj2wbSOhD0atDfV8yx-e8nWzmqwAs
+
+* How do you verify that PyPI can be trusted? 
+  https://snarky.ca/how-do-you-verify-pypi-can-be-trusted/
+
+* Overusing list comprehensions and generator expressions in Python 
+  https://treyhunner.com/2019/03/abusing-and-overusing-list-comprehensions-in-python
+
+## Projects & libraries to check out
+
+* Cookiecutter Template for realworld app based on Pyramid by Mandar Vaze 
+  https://github.com/mandarvaze/cookiecutter-pyramid-realworld
+
+* Lightweight pastebin for home network which may or may not be connected to the internet by Girish Joshi 
+  https://github.com/girish946/local-paste
+
+## Upcoming events in & around Pune
+
+* PyCon India 2019 on 12th & 13th Oct, 2019 
+  https://in.pycon.org/2019/
+
+## Thanks
+
+* Thanks to shoptimize for providing venue for Sept pythonpune meetup 2019 
```

Comment:
We should make it 'Sep' and 'PythonPune'

Reply:
Done